docs/changehistory/NextVersion.md

```diff
@@ -7,7 +7,10 @@ Table of contents:
 
 - [Geometry](#geometry)
   - [Clip any curve](#clip-any-curve)
-- [ECSQL instance properties](#ecsql-instance-properties)
+- [ECSql enhancements](#ecsq- enhancements)
```

Comment:
these changes should be in 4.3.0.md that was merged into master earlier today

Reply:
done.